omnibridge/wrappers/wrapper_interfaces/ModelWrapper.py

```diff
@@ -8,3 +8,11 @@ class ModelWrapper(JsonConvertable):
     @abstractmethod
     def process(self, model_input: ModelIO) -> ModelIO:
         pass
+    
+    @abstractmethod
+    def get_class_type_field(cls) -> str:
+        pass
+
+    @classmethod
+    def get_description(cls) -> str:
+        return 'Description not provided'
```

Comment:
IMO the `get_description` function doesn't fit with the wrapper classes as their main goals is to wrap an API call and process the response.

I think a mapping from wrapper to description would be enough, WDYT? (we can place it under the `cli` directory where it fits better logically)

Reply:
My idea was that when someone adds a wrapper, a description of its functionality would be provided. 
This will allow to propagate it to downstream apps such as flows and models.

Reply:
Got it, then please modify the decorator to `@abstractmethod` and instead of returning 'Description not provided', use `raise NotImplementedError()`, as `ModelWrapper` wouldn't be instantiated on its own.

`@classmethod` is usually used for constructing the class object.